src/main/java/io/dev/jobprep/domain/security/jwt/exception/TokenCachingException.java

```diff
@@ -0,0 +1,10 @@
+package io.dev.jobprep.domain.security.jwt.exception;
+
+import io.dev.jobprep.exception.code.ErrorCode;
+import io.dev.jobprep.exception.exception_class.CustomException;
+
+public class TokenCachingException extends CustomException {
```

Comment:
개인적인 생각인데, TOKEN 관련된 에러를 처리하는 CustomException을 하나 두고, 안에 넣을 `custom error code` 로 좀 더 세부적으로 나누도록 하는 게 어떨까요? 클래스가 너무 많아지면 또 관리하기 힘드니까요~

```suggestion
public class TokenException extends CustomException {
```